src/Sentry.AspNetCore.Grpc/SentryBuilderExtensions.cs

```diff
@@ -24,5 +28,29 @@ public static ISentryBuilder AddGrpc(this ISentryBuilder builder)
 
             return builder;
         }
+
+
+        private class SentryGrpcEventProcessor : ISentryEventProcessor
+        {
+            private static readonly SdkVersion NameAndVersion
+                = typeof(SentryGrpcInterceptor).Assembly.GetNameAndVersion();
+
+            private static readonly string ProtocolPackageName = "nuget:" + NameAndVersion.Name;
+            private const string SdkName = "sentry.dotnet.aspnetcore.grpc";
+
+            public SentryEvent Process(SentryEvent @event)
+            {
+                // Take over the SDK name since this wraps ASP.NET Core
+                @event.Sdk.Name = SdkName;
```

Comment:
THe only change that should have side affect. The rest should be small refactors